code/modules/client/preferences.dm

```diff
@@ -47,7 +47,6 @@ var/list/preferences_datums = list()
 	var/be_random_body = 0				//whether we'll have a random body every round
 	var/gender = MALE					//gender of character (well duh)
 	var/age = 30						//age of character
-	var/blood_type = "A+"				//blood type (not-chooseable)

```

Comment:
blood type no longer appears in the player window during lobby, we couldn't even choose the blood type before and having this as preference would've made my changes harder to implement. so bye bye